sections/getting-started.md

```diff
@@ -0,0 +1,156 @@
+    runHeaderBidding(unmatched)
+  })
+
+  // Setup Google Publisher Tag
+  var googletag = window.googletag || {}
+  googletag.cmd = window.googletag.cmd || []
+
+  googletag.cmd.push(() => {
+    adUnits.forEach(({ code, sizes, div }) => {
+      googletag
+        .defineSlot(code, sizes, div)
+        .addService(googletag.pubads())
+    })
+    googletag.pubads().disableInitialLoad()
+    googletag.pubads().enableSingleRequest()
+    googletag.enableServices()
+  })
+
+  function runHeaderBidding(units) {
+    // Header bidding logic
+  }
+</script>
+```
+
+> ℹ️ There are lots of valid ways to configure your ad stack; the setup above is for demonstration purposes.
+
+### Google Ad Manager
+
+In the interest of brevity this walkthrough overlooks many values you'll want to set in practice: rate, frequency capping, etc.
+
+1. Create a **Sponsorship** line item to prematch.
+
+  ![line item type](./assets/getting-started-line-item-type.png)
+
+1. Set it's goal to **100%**.
+
+  ![delivery goal](./assets/getting-started-delivery-goal.png)
+
+1. Set inventory targeting to target your desired inventory.
+
+  ![inventroy targeting](./assets/getting-started-target-inventory.png)
+
+1. Target a custom key value to represent when this line item should fire. For example below we're activating this line item when the key `gz` is present with a value of `true`.
+
+  ![custom targeting](./assets/getting-started-target-custom.png)
+
+1. Add creatives to your line item to match your inventory.
+
+1. Enable your line item.
+
+### Glimpse Zero Portal
+
+Our self-service portal and automatic GAM syncing isn't ready for prime time yet. So for now you'll need to provide us with your **Sponsorship** details so we can load it into our system. Generally we'll need to know:
+
+- When it starts and ends
+- Blackout or day parting details
+- Frequency capping details
+- What slots it targets
+- What custom targeting key value you've used
+
+
+### Testing
+
+And that's it 🥳 However, just to be safe it's always a good idea to test. It may take a few minutes for the line item to go live. Once it does you can enable debugging by running `localStorage.debug="glimpse*"` and should see the following process on a page:
+
+1. On the first load zero.js realises it has no targeting loaded and passes all units through to your `runActions` setup.
+2. In the background zero.js will fetch targeting information from our servers and save it to the local storage key `gp_zjs_config`.
+3. On a subsequent load zero.js will divide your units into matched and unmatched. It will tag matched units with the provided key values and pass the unmatched units, if any, through to `runAuctions`.
+4. When runActions completes GPT will tag slots and send an add call to Google Ad Manager.
+5. The response should have your Sponsorship line item against the matched slot and the other line items should fire as you would expect them to (for example triggering a Prebid line item)
+
```

Comment:
Pubs are calling out the need to not impact page load speeds. Is there a way this can give advice on how to prove this through the testing process?

Reply:
I'm gonna punt this for now since i don't have a clear answer. See slack thread with @perkyviking on measurement.

I think our demo page will go a long way to showing how this improves performacne but we'll need to do a bit more thinking about how we show that to pubs since there are soo many variables at play.